sentences/ca/_common.yaml

```
@@ -102,6 +102,12 @@ lists:
         out: locked
       - in: desbloquejat
         out: unlocked
+  presence_states:
+    values:
+      - in: home
+        out: a casa
+      - in: away
+        out: fora de casa
```

These should be the other way around, but keep in mind that `not_home` means "away", not "not home", i.e. out of any known zone

```suggestion
      - in: a casa
        out: home
      - in: fora de casa
        out: not_home
```

---

Actually, I'm not sure why this is in 'ca' language, but it's from pt-br language and it's my fault, I've inverted words there. Should I place a new PR or you just changed there?
Your sugestion are almost right in portuguese:
```
      - in: home
        out: casa
      - in: away
        out: fora_de_casa
```

If I'm right, `in` means the original word and `out` is translated word, right?
casa => home
fora_de_casa => not in home.

**_away_** means **_far away_** in portuguese.

---

I don't understand how the Catalan language leader could have committed something in the `/ca/` folder(s) as a result of this being for `pt-BR`. A quick Google Translate states that "a casa" and "fora de casa" are correct in Catalan as well.

@duhow ?

---

In a generic way, I can say "out" `fora` to mean "outside but without knowing where". with no context, assuming it is specific to home, it is "out of home".
"away" can be `lluny` as "far" but again missing the context, so i'd keep this iteration first to see impressions.